client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -267,7 +267,7 @@ const InvestigatedPersonInfo = (props: Props) => {
                         <>
                             <InfoItemWithIcon testId='age' name='גיל' value={+age < 1 ? 'פחות משנה' : age} icon={CakeOutlined} />
                             {
-                                +age <= maxComplexityAge && <ComplexityIcon tooltipText='המאומת מתחת לגיל 15' />
+                                +age <= maxComplexityAge && <ComplexityIcon tooltipText='המאומת מתחת לגיל 14' />
```

Comment:
Isn't there an existing PR w/ this change?

Reply:
you're right, I got out of this branch by mistake

Reply:
is it resolved?

Reply:
yes uploaded to dev in different branch